workflow-service/src/main/java/com/redhat/parodos/security/LoginController.java

```diff
@@ -39,13 +44,25 @@
 @Validated
 @RequestMapping("/api/v1/login")
 @Tag(name = "Login", description = "Login endpoint")
+@Slf4j
 public class LoginController {
 
+	private final UserService userService;
+
+	public LoginController(UserService userService) {
+		this.userService = userService;
+	}
+
 	@Operation(summary = "Login")
 	@ApiResponses(value = { @ApiResponse(responseCode = "200", description = "Succeeded", content = @Content),
 			@ApiResponse(responseCode = "401", description = "Unauthorized", content = @Content) })
 	@GetMapping
 	public ResponseEntity<String> login() {
+		String username = SecurityUtils.getUsername();
+		if (username != null && userService.findAllUserEntitiesByUsernameIn(List.of(username)).isEmpty()) {
+			userService.save(User.builder().username(username).enabled(true).build());
```

Comment:
Can we get additional info like first name, last name and email?

Reply:
I can add that from ldap for now